apis/apps/v1alpha1/backuppolicytemplate_types.go

```diff
@@ -236,6 +236,11 @@ type TargetInstance struct {
 	//
 	// +optional
 	ConnectionCredentialKey ConnectionCredentialKey `json:"connectionCredentialKey,omitempty"`
+
+	// Specifies the container port in the target pod.
```

Comment:
+ // If not specified, the first container and its first port will be used.

`json:",inline"` ?

Reply:
I think `json:"containerPort,omitempty"` is more proper, in which case the two subfields will be aggregated.